doc/data/messages/u/using-f-string-in-unsupported-version/details.rst

```diff
@@ -1 +1 @@
-You can help us make the doc better `by contributing <https://github.com/PyCQA/pylint/issues/5953>`_ !
+You should upgrade to python 3.6 to use f-strings. Even python 3.6 is past end of life at this point.
```

Comment:
```suggestion
f-strings were introduced in Python version 3.6; to use them, please use a more recent version of Python.
```

Perhaps I'm being pedantic but it sounds a bit like saying you must use Python 3.6 in order to use f-strings. I'm suggesting something more general but feel free to disregard.

Reply:
Oh timing! I see its merged just this minute - well no worries :D

Reply:
Huh, my bad it was auto-merge enabled. I shouldn't do that as it prevent approving with an optional suggestion (something I often do myself).

Reply:
Do you want to open a MR for your suggestion ? I think it's a lot better too.

Reply:
I'm happy either way; what you have already isn't wrong at all. Sure I'll create a quick follow-up 👨‍💻

Reply:
https://github.com/PyCQA/pylint/pull/8180